packages/code-snippet-block/src/settings.ts

```diff
@@ -206,56 +206,133 @@ export const settings: BlockSettings = {
             type: 'switch',
             defaultValue: true,
             label: 'Border',
-        },
-        {
-            id: 'border',
-            type: 'multiInput',
-            lastItemFullWidth: true,
-            show: (bundle: Bundle) => bundle.getBlock('withBorder')?.value === true,
-            onChange: (bundle: Bundle): void => {
-                appendUnit(bundle, 'lineWidth');
-            },
-            blocks: [
+            on: [
                 {
-                    id: 'lineStyle',
-                    type: 'dropdown',
-                    defaultValue: 'solid',
-                    choices: [
+                    id: 'border',
+                    type: 'multiInput',
+                    lastItemFullWidth: true,
+                    onChange: (bundle: Bundle): void => {
+                        appendUnit(bundle, 'lineWidth');
+                    },
+                    blocks: [
                         {
-                            value: 'none',
-                            label: 'None',
+                            id: 'lineStyle',
+                            type: 'dropdown',
+                            defaultValue: 'solid',
+                            choices: [
+                                {
+                                    value: 'none',
+                                    label: 'None',
+                                },
```

Comment:
Does is make sense to still have `None` as an option, now that we only show this setting when `widthBorder`-switch is `true`?

<img width="306" alt="Bildschirmfoto 2022-09-21 um 09 34 19" src="https://user-images.githubusercontent.com/11270687/191443597-a0a0f977-4d5e-4546-aa99-e7bee6c985a4.png">

Reply:
Yeah, good one. I don't think it's required in that case.